arch/riscv/boot/dts/bouffalolab/bl808.dtsi

```diff
@@ -61,15 +61,15 @@
 		#size-cells = <1>;
 
 		uart0: serial@30002000 {
-			compatible = "bouffalolab,uart";
+			compatible = "bflb,bl808-uart";
```

Comment:
Isn't this incompatible with the upstream bindings the Bouffalo kernel maintainer posted?

https://github.com/torvalds/linux/commit/d54c88d19380488afecbf0257fea7f5adb60ed7f

Reply:
As I understand they never made it to mainline. Plus I'm trying to integrate the latest opensbi with smaeul's patches here where he uses bl808-uart. 

https://github.com/smaeul/opensbi/commit/1767f7f5473b608cf8cfc66c5799a8f61c72c16d